en/patterns/scheduler.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Scheduler - Design System Pattern
+_description: The Scheduler Pattern Symbols is a complex combination of various components to provide meaningful application scenarios.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Scheduler
+
+Use the Scheduler Symbol as presets to display Scheduler as part of project management, task management, meeting planner or e-mail application.
```

Comment:
"as a preset" sounds better and maybe we should mention that this is just an interface illustrating a scheduler without any of the scheduling functionality that is also necessary.